addons/sourcemod/scripting/shop/player_manager.sp

```diff
@@ -1007,7 +1007,7 @@ void PlayerManager_DatabaseClear()
 	}
 }
 
-bool PlayerManager_IsInGame(int player_id)
+stock bool PlayerManager_IsInGame(int player_id)
```

Comment:
I do not think, that `stock` is needed here. This function used all time.

Reply:
![default](https://user-images.githubusercontent.com/12576822/36490942-f7c409f6-1742-11e8-85bd-ffd64b8dc986.png)

Reply:
Hmm... strange, maybe delete it, if it is **never used**

Reply:
There are no warnings when i'm trying to compiling it on master branch. Also builder said, that all fine, when compiling it.

Reply:
This was used [here](https://github.com/R1KO/Shop-Core/blob/master/addons/sourcemod/scripting/shop/db.sp#L200), but deleted in this PR.